src/Pages/Details.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import CandleDetails from '../components/CandleDetails';
+import StockScreener from '../components/StockScreener';
+
+const Details = () => {
+  const navigate = useNavigate();
+
+  return (
+
+    <Container>
+      <Button
+        style={{ marginTop: '2rem' }}
+        variant="outline-primary"
+        onClick={() => navigate('/')}
+      >
+        back
+      </Button>
+      <CandleDetails />
+      <StockScreener />
+    </Container>
```

Comment:
- [ ] [optional] @Paulinelali Great work implementing the changes requested, it would be nice if your detailed page used the same color as the home page, you can try and use slightly different shades like in the original design, this will ensure your design matches the given design.
NB `I loved your implementation of the details page its impressive.